migrations/V94__Update_General_Characteristics_AMSModelID_DemoCode.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE plan_general_characteristics
+  ADD COLUMN demo_code zero_string,
+  ADD COLUMN ams_model_id uuid;

```

Comment:
Do we know if the AMS Model ID is a UUID? I'm not sure we know the format of this quite yet, and it might make sense to just have this be `zero_string` as well, since it looks like the input type in GQL is already just `String`

Reply:
(Same for the Go struct type)